src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -25,7 +25,10 @@ object Run  {
 
   def apply(pipeline: Pipeline): Run = {
     val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
-    val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}]"
+    def tag: String = {
+      val shard = pipeline.key.index.map(x => s":$x").getOrElse("")
+      s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.scope.name}$shard]"
+    }
```

Comment:
It looks like the whole thing has changed so it's all academic but I completely disagree with the "only used in one place" argument for these things. Realizing that it's partially subjective and partially cultural, the point isn't abstracting to put things in one place but to enhance reasonability